kubernetes-client/src/test/java/io/fabric8/kubernetes/client/utils/SerializationTest.java

```diff
@@ -162,7 +162,7 @@ void testSerializeYamlWithAlias() {
       .isNotNull()
       .hasFieldOrPropertyWithValue("metadata.name", "test-pod-with-alias")
       .hasFieldOrPropertyWithValue("spec.nodeSelector.workload", "build")
-      .hasFieldOrPropertyWithValue("spec.tolerations.size", 1)
+      .returns(1, p -> p.getSpec().getTolerations().size())
```

Comment:
why is this change required?

Reply:
I think that in Java 17 AssertJ can't use reflection to access/invoke the list size method.